### ISSUE TITLE
`kele-get`: Start proxy asynchronously while accepting user input

### ISSUE THREAD

**Issue by jinnovation** [jinnovation/kele.el#72]
`kele-get` as introduced in #70 hangs prior to namespace completion, as Kele needs to populate the namespace cache by: 
1. Starting the proxy server process; 
2. Waiting for the proxy server to become "ready"; 
3. Querying the proxy server for namespaces; and **then**
4. Presenting namespace completion candidates to the user.

Of these steps, 1. and 2. are by far the most "expensive." They are also the easiest to parallelize.

We should start the proxy server **asynchronously** as soon as a user enters `kele-get`; pass the future to `kele--fetch-namespaces`; and wait on that future only when it actually becomes time to fetch namespaces from the API.

**Comment by jinnovation** [comment]
Now that we have `kele-resource` set up as a Transient prefix, this is something we could likely fold into the Transient prefix instantiation process, e.g. kick off proxy server creation async on prefix invocation and attach it to the prefix scope.

I see value in implementing this as a subclass of `transient-prefix`, e.g. `transient-prefix-with-proxy`.
